src/components/calcite-icon/calcite-icon.tsx

```diff
@@ -160,7 +160,6 @@ export class CalciteIcon {
 
   @Watch("icon")
   @Watch("filled")
-  @Watch("size")
```

Comment:
don't you need to change this to `scale` rather than removing?

Reply:
D-oh! Yes, good catch. I'll get that fixed right away.

Reply:
💥

Reply:
💥